packages/e2e/local-network/scripts/make-babbage.sh

```diff
@@ -107,7 +107,6 @@ cp templates/babbage/byron-configuration.yaml "${ROOT}/configuration.yaml"
 
 $SED -i "${ROOT}/configuration.yaml" \
      -e 's/Protocol: RealPBFT/Protocol: Cardano/' \
-     -e 's/minSeverity: Info/minSeverity: Debug/' \
```

Comment:
I guess this is the reason why I wasn't able to set `Info` log level by changing the `node-config.json`!